File: DSA/LeetCode/Arrays/1672. Richest Customer Wealth/Solution.py
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        max_wealth = float("-inf")

        for i in accounts:
            cust_sum = sum(i)

            if cust_sum > max_wealth:
                max_wealth = cust_sum

        return max_wealth

File: DSA/LeetCode/Arrays/1672. Richest Customer Wealth/Solution.js
/**
 * @param {number[][]} accounts
 * @return {number}
 */
var maximumWealth = function (accounts) {
  let max_wealth = Number.NEGATIVE_INFINITY

  for (let i = 0; i < accounts.length; i++) {
    const sum = accounts[i].reduce((acc, curr) => acc + curr, 0);

    if (max_wealth < sum) {
      max_wealth = sum
    }
  }
  return max_wealth
};